src/component/registry/css-registry.ts

```diff
@@ -13,21 +13,44 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+import { arraysAreIdentical } from '../helpers/array-utils';
+
 export class CssRegistry {
   private classNamesByBPMNId = new Map<string, string[]>();
 
+  /**
+   * Get the CSS class names for a specific HTML element
+   *
+   * @param bpmnElementId the BPMN id of the HTML element from the DOM
+   * @return the registered CSS class names
+   */
   getClassNames(bpmnElementId: string): string[] {
     return this.classNamesByBPMNId.get(bpmnElementId) || [];
   }
 
-  addClassNames(bpmnElementId: string, classNames: string[]): void {
-    // TODO: return the real modified id
-
+  /**
+   * Register the CSS class names for a specific HTML element
+   *
+   * @param bpmnElementId the BPMN id of the HTML element from the DOM
+   * @param classNames the CSS class names to register
+   * @return true if at least one class name from parameters was not registered; false otherwise
```

Comment:
⚠️ pretty unclear. We want to trigger an rendering change on actual css class change.
- for a single class: if the class has been added
- for several classes: at least when one has been added

So for the test name and this doc, this will be more straightforward to use a positive/affirmative form like
```javascript
/**
 *@return true if at least one class name from parameters has been added; false otherwise
 */
```